lib/realtime_web/channels/realtime_channel/presence_handler.ex

```diff
@@ -17,23 +18,28 @@ defmodule RealtimeWeb.RealtimeChannel.PresenceHandler do
   alias RealtimeWeb.Presence
   alias RealtimeWeb.RealtimeChannel.Logging
 
-  @spec call(map(), Phoenix.Socket.t()) ::
+  @spec call(map() | :sync_presence, Phoenix.Socket.t()) ::
           {:noreply, Phoenix.Socket.t()} | {:reply, :error | :ok, Phoenix.Socket.t()}
-  def call(
-        %{"event" => event} = payload,
-        %{assigns: %{is_new_api: true, presence_key: _, tenant_topic: _}} = socket
-      ) do
-    socket = count(socket)
-    {result, socket} = handle_presence_event(event, payload, socket)
+  def call(%{"event" => event} = payload, socket) do
+    event = String.downcase(event)
```

Comment:
For performance, you may do the following if applicable:

```suggestion
    event = String.downcase(event, :ascii)
```